src/union.ts

```diff
@@ -48,60 +55,46 @@ const createFSProxy = (watchers: FSWatcher[]) =>
     },
   );
 
-const fsPromisesMethods = [
-  'access',
-  'copyFile',
-  'open',
-  'opendir',
-  'rename',
-  'truncate',
-  'rmdir',
-  'mkdir',
-  'readdir',
-  'readlink',
-  'symlink',
-  'lstat',
-  'stat',
-  'link',
-  'unlink',
-  'chmod',
-  'lchmod',
-  'lchown',
-  'chown',
-  'utimes',
-  'realpath',
-  'mkdtemp',
-  'writeFile',
-  'appendFile',
-  'readFile',
-] as const;
+export type VolOptions = {
+  readable?: boolean;
+  writeable?: boolean;
```

Comment:
This should be spelt without an 'e': `writable`.

Reply:
I take it you're American. It's a valid spelling. I can change it (everywhere) if you insist..

Reply:
Actually I'm a Kiwi :)

The reason I ask for it to be spelt without an 'e' is for consistency: Node uses [`writable`](https://nodejs.org/api/stream.html#stream_writable_streams).

Reply:
haha if i'd have known you were kiwi i'd have have definitely gone with   ".. you're american 🙄 "

I'll update